taxcalc/parameters.py

```diff
@@ -33,24 +37,36 @@ class CompatibleDataSchema(ma.Schema):
 
 class Parameters(pt.Parameters):
     """
-    Base Parameters class that wraps ParamTools, providing parameter indexing
-    for tax policy in the adjust method and backwards-compatible preserving
-    layer that supports Tax-Calculator's conventional reform formatting style
-    as well as convenience methods like set_Year for classes operating on this
-    one.
+    Base class that wraps ParamTools, providing parameter indexing
+    for tax policy in the ``adjust`` method and convenience methods
+    like ``set_year`` for classes inheriting from it. It also provides
+    a backwards-compatible layer for Tax-Calculator versions prior to 3.0.
```

Comment:
I rephrased this some to (hopefully) provide more clarity and context, but the substance should be the same.